openshift/system/config/thinking_sphinx.yml

```diff
@@ -0,0 +1,19 @@
+production:
```

Comment:
This and `unicorn.rb` are the only files from `config/docker` that were not overriden in `openshift/system/config`, so copying them here.